libraries/rush-lib/src/api/RushConfiguration.ts

```diff
@@ -382,13 +353,6 @@ export class RushConfiguration {
    */
   public readonly packageManagerToolVersion: string;
 
-  /**
-   * The absolute path to the locally package manager tool.  If "rush install" has not
-   * been run, then this file may not exist yet.
-   * Example: `C:\MyRepo\common\temp\npm-local\node_modules\.bin\npm`
-   */
-  public readonly packageManagerToolFilename: string;
```

Comment:
Why was `packageManagerToolFilename` removed?

Reply:
Added this back